glide.lock

```diff
@@ -20,7 +20,7 @@ imports:
   subpackages:
   - edwards25519
 - name: github.com/FactomProject/factom
-  version: 201a384d8e70e8d81b53c730b5432787cf4aeb24
+  version: 2403dbaeb4670238c5135ef76732cbc902140683
```

Comment:
Update this hash to the new one (you made a change).
Just manually update this hash, do not run a glide update.
Glide update updates all deps, which is not what we want.

It is a downside of glide, not being able to update 1 dep.